model/countries/NL/NL-formatting-rules.yaml

```diff
@@ -0,0 +1,77 @@
+formatting-rules:
+  address:
+  - street-address-alternative-1
+  - separator: "\n"
+  - postal-code
+  - separator: " "
+  - locality1
+  - separator: "\n"
+  - country-name
+  - skip: country  # redundant with country-name
+  - skip: street-address  # redundant with street-address-alternative-1
+  - skip: admin-area1 # not commonly used
```

Comment:
Could you add to the PR description the main sources that you used to build these expressions? I'm also interested in the following questions:

1. Is okay to always use `-` as the separator building and unit?
2. Is state (admin-area1) really not used in the Netherlands?